std-bits/base/src/main/java/org/enso/base/Time_Utils.java

```diff
@@ -103,7 +103,7 @@ public static String date_time_format(ZonedDateTime dateTime, Object format) {
     return DateTimeFormatter.ofPattern(format.toString()).format(dateTime);
   }
 
-  public static String date_time_format(ZonedDateTime dateTime, Object format, Locale locale) {
+  public static String date_time_format_with_locale(ZonedDateTime dateTime, Object format, Locale locale) {
```

Comment:
Good, but ofc I assumed this suggestion applies to all related functions, so ideally I'd split the overrides of `time_of_day_format` and `local_date_format` into separate functions too.

Reply:
Yes, forgot.